atmos_model.F90

```diff
@@ -491,23 +490,21 @@ subroutine atmos_model_init (Atmos, Time_init, Time, Time_step)
   type (atmos_data_type), intent(inout) :: Atmos
   type (time_type), intent(in) :: Time_init, Time, Time_step
 !--- local variables ---
-  integer :: unit, ntdiag, ntfamily, i, j, k
-  integer :: mlon, mlat, nlon, nlat, nlev, sec, dt
+  integer :: unit, i
+  integer :: mlon, mlat, nlon, nlat, nlev, sec
   integer :: ierr, io, logunit
-  integer :: idx, tile_num
+  integer :: tile_num
   integer :: isc, iec, jsc, jec
-  integer :: isd, ied, jsd, jed
-  integer :: blk, ibs, ibe, jbs, jbe
   real(kind=GFS_kind_phys) :: dt_phys
-  real, allocatable    :: q(:,:,:,:), p_half(:,:,:)
-  character(len=80)    :: control
-  character(len=64)    :: filename, filename2, pelist_name
-  character(len=132)   :: text
-  logical              :: p_hydro, hydro, fexist
+#ifndef INTERNAL_FILE_NML
```

Comment:
I think dycore does not support this option any more. It has to use INTERNAL_FILE_NML even though the INTERNAL_FILE_NML are still in some of the code.

Reply:
Se are you saying we can remove this ifndef entirely? I still see a lot of similar ifdefs in ccpp and dycore

Reply:
Yes, GFDL will do some clean up. The code only works with INTERNAL_FILE_NML defined.

Reply:
Removed in d5409df